README.md

```diff
@@ -56,7 +56,6 @@ cd gelato-kyber
 
 yarn install  # or npm install
 
-npx buidler compile
```

Comment:
They still need to `compile` we should move the `.env`  section before `setup` then.

Reply:
doesnt it automatically compile before running the first script? It did so in the previous version

Reply:
added after .env